pkg/controller/plan/adapter/openstack/builder.go

```diff
@@ -820,26 +819,75 @@ func (r *Builder) TemplateLabels(vmRef ref.Ref) (labels map[string]string, err e
 	default:
 		os = UnknownOS
 	}
+	return
+}
 
+func getPreferenceOs(os, version, distro string) string {
+	if os != UnknownOS && version != "" {
+		if distro == CentOS && len(version) >= 1 && (version[:1] == "8" || version[:1] == "9") {
+			os = fmt.Sprintf("%s.stream%s", distro, version)
+		} else if os == Windows {
```

Comment:
i do wonder if we should set them to include `virtio`. @ahadas wdyt?

Reply:
I don't see much sense in having this different from the logic we have for ovirt - after all, both platforms are based on the same hypervisor..